### PR TITLE
SEO-172844-Xamarin-weblink-validatetor-images

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Please review the changes have made in image alt text and image names.
Thanks.
Regards,
Edith.

### CODE REVIEW

File: Xamarin/Visual-Studio-Integration/Download-and-Installation.md [hunk 1]
```diff
@@ -25,16 +25,16 @@ The steps below assist you to how to install the Syncfusion Xamarin extensions f
 	> In Visual Studio 2017 or lower, go to Tools -> Extensions and Updates
 3.	On the left, click the **Online** tab and type **"Xamarin Extensions"** in the search box.
 
-	![Manage Extension](Download-and-Installation-images/ManageExtension.PNG)
+	![Manage Extension in Xamarin Visual studio](download-and-installation_images/xamarin-manage-extension.PNG)
```
Comment (MallikaRK) [inline Xamarin/Visual-Studio-Integration/Download-and-Installation.md:28]: Check the image folder name again. 
Why you used underscore in image folder name?

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Visual-Studio-Integration/Download-and-Installation.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/Visual-Studio-Integration/Download-and-Installation.md:28]: resolved

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**